megamek/src/megamek/client/Client.java

```diff
@@ -1757,4 +1884,18 @@ public void setCurrentHex(IHex hex) {
     public void setCurrentHex(Coords hex) {
         currentHex = hex;
     }
+    
+    /** Returns true when the player is a bot added/controlled by this client. */
+    public boolean isLocalBot(IPlayer player) {
+        return bots.containsKey(player.getName());
+    }
+    
+    /** Returns true when the player is a bot added/controlled by this client. */
```

Comment:
Comment doesn't match method behavior. Also, do we really want to be using exceptions as flow control?

Reply:
I'll change it to simply return null when the given player isn't actually a bot. Saves space. I don't mind but for the sake of clarity: When a method can't return a value that is correct, then why not use an exception? It just means it'll fail clearly when the caller doesn't do the appropriate checks before. In the same case, returning null will likely lead to an NPE, just at another place. I wouldn't know what else to return.

Reply:
Looking at it again, in this particular case, I don't think I have a problem with the exception, just the mismatched comment. Maybe include an update to the comment that indicates this throws an exception.